modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -2434,12 +2434,25 @@ definition {
 	}
 
 	@description = "Verify that, when the user deletes a translation, the action reflects in the language selector"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test DeletedTranslationsAreNotDisplayedOnLanguageSelector {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#DeletedTranslationsAreNotDisplayedOnLanguageSelector";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.viewTranslationDropdown(translationOptionSelected = "en-us");
+
```

Comment:
There is a step missing, `viewTranslationLanguageOptionPresent`, you could see if the translation language option that you are going to delete, it's present before it's deleted.